### ISSUE TITLE
Can't find variable: process after `v3.66.0` update

### ISSUE THREAD

**Issue by corygibbons** [sanity-io/sanity#7964]
**Describe the bug**

After updating to `v3.66.0` I'm getting a `ReferenceError: Can't find variable: process` error in my embedded studio. Similar setup to: https://github.com/SimeonGriggs/sanity-react-router-template

This seems to be the offending line: https://github.com/sanity-io/sanity/blob/aeff5521ef6c5e440baf8bfd31349d0a823c74cd/packages/sanity/src/core/studio/StudioTelemetryProvider.tsx#L30

**Expected behavior**

No error 😢

**Which versions of Sanity are you using?**

```
@sanity/cli (global)         3.66.0 (up to date)
@sanity/preview-url-secret    2.0.5 (up to date)
@sanity/react-loader        1.10.28 (up to date)
@sanity/vision               3.66.0 (up to date)
@sanity/visual-editing       2.10.4 (up to date)
sanity                       3.66.0 (up to date)
```

**Comment by thomasKn** [comment]
Same issue in my Hydrogen app.

**Comment by bjoerge** [comment]
Thanks for reporting! Fixed by #7967 – will cut a patch release shortly.

**Comment by bjoerge** [comment]
Fix released in [v3.66.1](https://github.com/sanity-io/sanity/releases/tag/v3.66.1) – thanks again for reporting!

**Comment by corygibbons** [comment]
@bjoerge Thanks for the quick fix!
